### DISCUSSION TITLE
Perdi o primeiro encontro, ainda dá tempo de participar?

### DISCUSSION THREAD

**Discussion by villares** [clube-de-leitura-python/pense-em-python-2021#8] [Perguntas & Respostas] [answered]
Tivemos um primeiro encontro em 30/07, foi bem rápido, conversamos um pouco sobre o prefácio, motivações para participar do grupo. Destacamos que seguimos o código de conduta da Python Brasil.

**Comment by villares** [comment, marked as answer]
Então ainda dá tempo de participar, das seguintes maneiras:
- Confira um pouco do que rolou de conversa [neste thread](https://github.com/clube-de-leitura-python/pense-em-python-2021/discussions/5)
- Leia os capítulos 1 e 2 do livro
E/OU
- Tente agitar um novo grupo de leitura!

**Reply by albanesere** [reply]
Olá! Agradeço tudo.
Eu gostaria de acompanhar com vcs, mas sexta feira estava desmontando a casa que foi abrigo na pands... eu estava numa vila de pesacadores em Ubatuba... e agora voltei pra civilizaão. Aqui em Sampa e já organizada com a vida de volta presencial na Universidade. Eu vou tentar ler para essa sexta, mas não sei se dou conta, essa semana é de planejamento. De todo jeito eu vou tentar tirar o atraso ate a segunda semana de agosto que estara tudo mais nos trilhos dos trens aqui do lado de fora da minha janela da civilização. :D Agradeço muito. Ab Regina

**Reply by villares** [reply]
Salve, Regina! O próximo encontro é dia 13/08 (sempre a cada duas semanas). Veja se dá pra ler e testar alguma coisinha!

**Comment by albanesere** [comment]
ebaaaa... ai da sim!
Grata e até la´
Regina

Em ter., 3 de ago. de 2021 às 14:46, Alexandre B A Villares <
***@***.***> escreveu:
